geomstats/information_geometry/multivariate_normal.py

```diff
@@ -9,13 +9,137 @@
 
 import geomstats
 import geomstats.backend as gs
+import geomstats.errors as errors
 from geomstats.geometry.base import OpenSet
 from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.product_manifold import ProductManifold
+from geomstats.geometry.product_riemannian_metric import NFoldMetric
 from geomstats.geometry.riemannian_metric import RiemannianMetric
+from geomstats.geometry.spd_matrices import SPDAffineMetric, SPDMatrices
 from geomstats.information_geometry.base import InformationManifoldMixin
 from geomstats.information_geometry.normal import NormalMetric
 
 
+class MultivariateNormalDistributions:
+    """Class for the multivariate normal distributions.
+
+    This class is a common interface to the following different situations:
```

Comment:
Nice!

Reply:
- Should the interface just be called "NormalDistributions"
-  and the univariate normals become "UnivariateNormalDistributions"
- and when n = 1 then UnivariateNormalDistributions is called?
It'd be nicer from a user perspective.